src/s2/s2builder.h

```diff
@@ -1303,7 +1303,7 @@ inline void S2Builder::AddPoint(const S2Point& v) {
 }
 
 inline int S2Builder::num_input_edges() const {
-  return input_edges_.size();
+  return static_cast<int>(input_edges_.size());
```

Comment:
Is this to fix an error or warning?

Reply:
that may have been a warning.  I can revert it.

Reply:
verified it was only a warning.

Reply:
Let's just keep this to fixing breakages.

Reply:
I already reverted this in a prior commit, should have specified in my comment.